src/test/resources/expected_helm_output/bitbucket/output.yaml

```diff
@@ -39,6 +39,7 @@ metadata:
     app.kubernetes.io/instance: unittest-bitbucket
     app.kubernetes.io/version: "7.12.1-jdk11"
     app.kubernetes.io/managed-by: Helm
+  annotations:
```

Comment:
if there is no annotation to set then good to have `annotations: {}`

Reply:
this is for the expected output though, right? I can't really control that with the values